routes/web.php

```diff
@@ -30,5 +30,12 @@
 Route::middleware('auth')->group(function () {
 
     Route::get('/dashboard', [DashboardController::class, 'index'])->name('dashboard');
+    Route::post('/reset-password', [App\Http\Controllers\Auth\NewPasswordController::class, 'store'])
+        ->name('password.reset');
+    Route::post('/forgot-password', [App\Http\Controllers\Auth\PasswordResetLinkController::class, 'store'])
+        ->name('password.email');
```

Comment:
Please add api. prefix to differentiate from the main platform's routes

Reply:
Also, move these routes to api.php since these will be mainly for the Admin Panel RESTful API.